assets/wire/reducers.js

```diff
@@ -61,8 +61,9 @@ const initialState = {
     newItemsByTopic: {},
     readItems: {},
     wire: {
-        services: [],
-        activeService: {},
+        navigations: [],
+        activeProducts: {},
+        activeNavigation: {},
```

Comment:
I think there will be only 1 active nav at a time so we can simplify the state, we were testing multiple products active but it didn't work out